README.md

```diff
@@ -18,9 +18,9 @@ If you want to configure additional connectors, you can pass the catalog configu
 > " 'cosmosdb': ['connector.name=mongodb','mongodb.seeds=test.documents.azure.com:10255','mongodb.credentials=testuser:secretpassword@prestocollection','mongodb.ssl.enabled=true'],'sqlserver': ['connector.name=sqlserver','connection-url=jdbc:sqlserver://testsqlserver.database.windows.net:1433;database=testdb;user=testuser@testserver;password=secretpassword;encrypt=true;trustServerCertificate=false;hostNameInCertificate=*.database.windows.net;loginTimeout=30;', 'connection-user=testuser','connection-password=secretpassword'] "
 
 # Airpal
-To optinally install [airpal](https://github.com/airbnb/airpal), 
+To optionally install [airpal](https://github.com/airbnb/airpal), 
 
-1. SSH to the cluster and run the following command to know address of the presto server
+1. SSH to the cluster and run the following command to known address of the presto server
```

Comment:
Actually the "SSH to the cluster and run the following command *to know* address of the presto server" is correct since we are trying to determine the presto server address with this command.